### ISSUE TITLE
fine tuning using my own dataset

### ISSUE THREAD

**Issue by srihima** [kenshohara/3D-ResNets-PyTorch#71]
sir i m trying to finetune the given code on my own dataset so how should i proceed like should label my videos and get annotations ?

**Comment by Henry-E** [comment]
They have good examples of what format to use. They're able to run on three different tasks, using the same format for all of them. https://github.com/kenshohara/3D-ResNets-PyTorch/tree/master/utils . I would say copy the scripts. They also mention the assumed data structure of the inputs. https://github.com/kenshohara/3D-ResNets-PyTorch#running-the-code It all seems pretty nicely generic for running your own fine tuning.

**Comment by srihima** [comment]
thank you sir i solved it..

**Comment by Fazlik995** [comment]
@srihima could you say me how do you solve your problem, please?

I also want to fine-tune custom dataset. 

Could you tell me, please, how did you get annotation file for your dataset?

Thank you.
